docs/authoring/working_with_teams.md

```diff
@@ -0,0 +1,514 @@
+draft of the tool that completes every question, even if it is in a very
+rough order. Set a deadline for this. With the help of the Weaver, it should 
+be possible to reach this in 10-15 hours for even a quite long form. Be
+willing to postpone certain features to hit this first milestone.
+
+You'll need to communicate very clearly with the client that this rough prototype is
+not even close to the final product but that you still need their high level feedback
+on it.
+
+For a project involving multiple forms, we recommend:
+
+* Starting with the simplest form to get the client's early feedback
+* Moving on to the most complex form to confirm the shape and logic
+  of the whole process with your client
+* Then continuing with the remaining forms
+
+:::tip Sample timeline for a project with 12 forms of varied complexity
+
+Each phase is scheduled for 2 weeks-2 months:
+1.	Phase 1, 2 weeks: Completion of a draft of 1 key document, which may include a subset of the full functionality, in order to obtain the client's input and review. The decisions that come out of this first document will guide completion of the remaining 11 documents.
+2.	Phase 2: Completion of drafts of up to 6 of the remaining documents.
+3.	Coordination of usability tests to identify changes that should apply to the remaining documents.
+4.	Phase 3: Completion of the full document set. Some functionality may be in a rough draft or ommitted for this stage.
+5.	Phase 4: All functionality is implemented. client to review and offer suggestions. Coordinate for second round of usability testing and feedback.
+6.	Final phase: based on the final feedback and proposed changes, complete all remaining work. Budget up to 30 hours for this phase.
+
+:::
+
+### Understanding success metrics
+
+Consider success in terms of:
+
+* Time
+* Total people helped
+* People who can now complete the process who couldn't before
+
+For a commercial project, this directly corresponds to money: either
+a reduced fee you can charge a client, or an increased profit, or both.
+
+1. How will we know if the project is successful?
+        * Time
+                * How much time does the process take now?
+                * What time savings (per use) is enough to justify the project?
+        * People helped
+                * How many people **need** the solution your tool helps with? For example: About 10,000 people are brought to eviction court in Massachusetts each year.
+                * How many people would be **enough** to reach? 10%? 30%? More? For example: helping 2,000 of the 10,000 people who are
+                evicted represents a significant percentage of the whole even though it is
+                a relatively small absolute number.
+        * Closing an access gap
+                * Who is not **appropriate** to complete the process now due to lack of training or legal specific expertise, but can be after your tool exists?
```

Comment:
It's something more complex than that, but I'm not sure I want to rephrase this more. The example covers it pretty well. A professional who isn't a lawyer is usually going to stay away from helping with something out of their expertise. Like I can change an outlet out, but I wouldn't help a stranger with that. I'd tell them to call an electrician.

A social worker can easily help someone complete a legal form they see every day, but they'll refuse to do so both because it's not the thing they're specifically trained in and because of concerns about unauthorized practice of law. 

However, if they help use the tool which itself gives the substantive help, they should be clear from concern about giving the wrong advice.

This is all too long for this bullet.